massgen/frontend/displays/rich_terminal_display.py

```diff
@@ -2668,6 +2676,9 @@ async def display_final_presentation(
             self._stored_presentation_agent = selected_agent
             self._stored_vote_results = vote_results
 
+        # Finalize the file
+        self._finalize_final_presentation_file(presentation_file_path)
```

Comment:
This is a good approach we can use later for coding agents to write answers in a file.